<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CAE9FF</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>441</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#e1f3ff</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Generar tratamiento</string>
    </attribute>
    <widget class="QLabel" name="label_nombre_paciente">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de paciente</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_paciente">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_tratamiento">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tratamiento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tratamiento">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_generar_tratamiento">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar nuevo tratamiento</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Revisar expediente</string>
    </attribute>
    <widget class="QLabel" name="label_nombre_paciente_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de paciente</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_paciente_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_buscar_paciente">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_expediente">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>381</width>
       <height>150</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Tratamiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de inicio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ãšltima cita</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_ver_tratamiento">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver tratamiento:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tratamiento">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dar_seguimiento">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>270</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Dar seguimiento</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
